spec/factories/courses.rb

```diff
@@ -1,10 +1,12 @@
 FactoryBot.define do
   factory :course do
-    name { "#{Faker::Job.unique.field} 101" }
+    association :institute
```

Comment:
This is identical to just `institute`. What you have is fine but just if you wanted a more shorthand way of doing it in future.

```suggestion
    institute
```